$NetBSD: patch-src_mesa_drivers_x11_Makefile.am,v 1.1 2019/01/19 21:54:03 tnn Exp $

We don't need libmesautil.la here. Those symbols seem to already be
included in libmesa.la.

--- src/mesa/drivers/x11/Makefile.am.orig	2019-01-17 11:26:23.000000000 +0000
+++ src/mesa/drivers/x11/Makefile.am
@@ -68,7 +68,6 @@ GL_PATCH = 0
 lib@GL_LIB@_la_LIBADD = \
 	$(top_builddir)/src/mesa/libmesa.la \
 	$(top_builddir)/src/mapi/glapi/libglapi.la \
-	$(top_builddir)/src/util/libmesautil.la \
 	$(SHARED_GLAPI_LIB) \
 	$(GL_LIB_DEPS)
 
